### DISCUSSION TITLE
layui里有没有支持makedown语法的组件？

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2105] [讨论]
因工作需要，需要在页面上显示makedown语法的文本（含图片）
请问layui里有没有现成的组件可以支持的？

**Comment by sentsim** [comment]
UI 组件一般不会内置 Markdown 解释器。
直接用 markdown-it 或 marked 即可，它们都是主流的 md 解释器

**Reply by HuangLei0x2e5** [reply]
好的谢谢

**Reply by qurge** [reply]
一直在找一个md的解析程序，谢谢大佬

**Comment by Laiyiwen-02** [comment]
可以用 editor.md 或者 marked.js 实现类似的功能，就拿 marked.js 来说（因为 editor.md 的 css 貌似会和 layui.css 冲突），对于一个已经准备好的 md 文本，其渲染如下：

```javascript
marked.parse(text)
```  

关于 marked.js 很多 csdn/cnblog 上面的大佬也介绍的很详细，对于上面那一段代码，text 是你准备好的文本，使用 marked 组件可以去官网获取引入方式或者看一些大佬的文章。
